### PR TITLE
Update Querying_an_Elasticsearch_database.md

### PR DESCRIPTION
**Pull request by daudahabbasi**
I have added the following section.

Manually setting the replication counts for indices

To get the relevent information on all indices and set replication count in the Elasticsearch database, enter the query below.

GET _cluster/health?level=shards&pretty

This will return the name of the indices, state of the indices and number of replica set as following.

  "indices" : {
    "dms-alarms-2022.06.11.05-000016" : {
      "status" : "green",
      "number_of_shards" : 3,
      "number_of_replicas" : 0,
      "active_primary_shards" : 3,
      "active_shards" : 3,
      "relocating_shards" : 0,
      "initializing_shards" : 0,
      "unassigned_shards" : 0,
      "shards" : {
        "0" :
    },
    "dms-alarms-2023.03.22.10-000063" : {
      "status" : "green",
      "number_of_shards" : 3,
      "number_of_replicas" : 0,
      "active_primary_shards" : 3,
      "active_shards" : 3,
      "relocating_shards" : 0,
      "initializing_shards" : 0,
      "unassigned_shards" : 0,
      "shards" : {
        "0" :
    },

To manually set the replica count to 2 use the following query.

PUT /<my-index>/_settings
{
  "index" :{
    "number_of_replica" : 2
  }
}

To set the replica count to 2 for all of the indices use the following query.

PUT /all/_settings
{
  "index" :{
    "number_of_replica" : 2
  }
}

### CODE REVIEW

**Comment by MariekeGO** [comment]
@SLC-Laurens-Vercruysse, could you confirm if this is OK to merge?

File: user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md [hunk 1]
```diff
@@ -46,6 +46,66 @@ This will return the name of the index, the state of the index, and its size.
...
+    },
+```
+
+You can then for example set the replica count to 2 for a specific index using the following query:
+
+```txt
+PUT /<my-index>/_settings
+{
+  "index" :{
+    "number_of_replica" : 2
```
Comment (SLC-Laurens-Vercruysse) [inline user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md:93]: Missing an s here: number_of_replica**s**

**Comment by MariekeGO** [comment]
@SLC-Laurens-Vercruysse, a number of changes have been implemented based on your feedback. Could you take another look?

File: user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md [hunk 2]
```diff
@@ -101,11 +106,13 @@ To set the replica count to 2 for all indices, use the following query:
...
 {
   "index" :{
-    "number_of_replica" : 2
+    "number_of_replicas" : 2
   }
 }
 ```
 
+Note: For ElasticSearch version 6.8.23 replace PUT /all/_settings with PUT /*/_settings as PUT /all/_settings does not work with ElasticSearch version 6.8.23
+
```
Comment (SLC-Laurens-Vercruysse) [inline user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md:116]: I'm not sure here which version does work with PUT /all/_settings since we only support Elastic 6.8.0. Was this tested in the first place? On which version?

File: user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md [hunk 3]
```diff
@@ -48,6 +48,11 @@ Especially the name and state of the index may be interesting for debugging purp
 
 ### Manually setting the replication count for indices
 
+Follosing commands allow you to alter the number of replicas (per shard) for indices matching the specific criteria. This is useful if you would like to reduce your per-node shard count and also if there are more replicas then the number of nodes, which is not very useful and put an unnecessary strain to the system. Furthermore you can use it to cutomize the availability of your ElasticSearch database.
```
Comment (SLC-Laurens-Vercruysse) [inline user-guide/Advanced_Functionality/Databases/Database_queries/Querying_an_Elasticsearch_database.md]: Several spelling mistakes. Would reword this entire paragraph e.g. as such: 

The following commands allow you to alter the number of replicas per shard. 
This would typically be used to configure redundancy and availability based on the amount of Elasticsearch nodes you have available.

!TIP 
If the amount of shards you have is too large, decreasing the amount of replicas per shard is one option to decrease the strain on the system.

**Review by SLC-Laurens-Vercruysse — CHANGES_REQUESTED** [review]
Still some things that can be slightly improved. See comments.

**Review by MariekeGO — APPROVED** [review]
